src/systems/filecoin_blockchain/storage_power_consensus/storage_power_consensus_subsystem.go

```diff
@@ -44,22 +44,20 @@ func (spc *StoragePowerConsensusSubsystem_I) IsWinningPartialTicket(stateTree st
 	return spc.ec().IsWinningChallengeTicket(challengeTicket, sectorUtilization, networkPower, sectorsSampled, numSectors)
 }
 
-// TODO: fix linking here
-var node node_base.FilecoinNode
-
 func (spc *StoragePowerConsensusSubsystem_I) _getStoragePowerActorState(stateTree stateTree.StateTree) StoragePowerActorState {
 	powerAddr := addr.StoragePowerActorAddr
 	actorState, ok := stateTree.GetActor(powerAddr)
 	util.Assert(ok)
 	substateCID := actorState.State()
 
-	substate, err := node.LocalGraph().Get(ipld.CID(substateCID))
-	if err != nil {
-		panic("TODO")
+	substate, ok := spc.node().Repository().StateStore().Get(ipld.CID(substateCID))
+	if !ok {
+		// TODO better err?
```

Comment:
panic is fine since we dont expect an error here when reading the state of the power actor.